common/models/transducer/alignment.py

```diff
@@ -1,16 +1,22 @@
 from returnn.tf.util.basic import get_shape_dim, check_input_dim
+from returnn.tf.util.data import Data
+from returnn.tf.layers.basic import LayerBase
+from tensorflow import Tensor
```

Comment:
Do not import `Tensor` directly.
Use `import tensorflow as tf` and then refer to it as `tf.Tensor`. (This is the standard way of doing it. See also all our other code.)